### ISSUE TITLE
SassError: Undefined variable.

### ISSUE THREAD

**Issue by RodrigoEnable** [telerik/kendo-themes#4284]
I'm upgrading Angular to version 15 and Kendo, I use the bootstrap theme and I get the following error.

![image](https://user-images.githubusercontent.com/62714522/218328130-d202ccb4-70fa-4b32-a17f-d4d28885e3f5.png)

OS: [Windows]
Node version: [14.21.1]
NPM version: [7.20.0]

**Comment by rramosgit** [comment]
I'm getting same error , did you find a solution? I started seeing the error once I changed kendo-theme-bootstrap from 5.5.0 to 6.2.0

**Comment by RodrigoEnable** [comment]
Yes, I solved it by overriding the property that caused the error in my style sheet.
![image](https://user-images.githubusercontent.com/62714522/235781486-ebebe53a-303e-4dc5-8e6e-516dcdfee366.png)
Note: I also needed to overwrite $wcag-min-contrast-ratio to eliminate the console error.

**Comment by rramosgit** [comment]
thanks for answer , I tried but didn't work, in my case, I tried to overwrite functions ,

![image](https://user-images.githubusercontent.com/13801644/235927539-ddc37f85-5b50-4c9b-ae76-62c5c7bbca2e.png)

**Comment by KimBum** [comment]
Can we bump this? Defining the variables is just a workaround. The files which to import from have moved to a different import approach. But if you import or even `@use` those, you would get util output for each of those cases right now.

**Comment by TeyaVes** [comment]
no longer reproducible as of Kendo Bootstrap v. 12.0.0
